src/imitation/util/networks.py

```diff
@@ -126,12 +129,12 @@ def update_stats(self, batch: th.Tensor) -> None:
         tot_count = self.count + batch_count
         self.running_mean += delta * batch_count / tot_count
 
-        self.running_var *= self.count
-        self.running_var += batch_var * batch_count
-        self.running_var += th.square(delta) * self.count * batch_count / tot_count
-        self.running_var /= tot_count
+        m_a = self.running_var * self.count
```

Comment:
@taufeeque9 does this rewrite look OK for you from a numerical stability perspective?

Reply:
The computation should be identical to the original, it's just that running_var is not updated in place. This shouldn't change the result unless I miss something?

Anyway, this change isn't necessary, it can be reverted.